client/src/api/PostAPI.ts

```diff
@@ -1,4 +1,5 @@
 import axios from 'axios';
+import { post } from 'cypress/types/jquery';
 import { MHJSON } from 'types/MHJSON';
 const SERVER_PORT = 3031;
```

Comment:
I'm not sure if this is how we get the base URL. @wilgao09? I also need to know because I have to use the fetch() API for Next.js server-side rendering.